### PR TITLE
lightning: record actual training step

### PR DESCRIPTION
**Pull request by dberenbaum**
Lightning was recording one step per logging iteration, but by default that logs every 50 steps (https://pytorch-lightning.readthedocs.io/en/stable/extensions/logging.html#logging-frequency). This leaves you with 3 different values between epoch, lightning step, and dvclive step.

### CODE REVIEW

**Review by daavoo — APPROVED** [review]

File: tests/test_lightning.py [hunk 1]
```diff
@@ -143,3 +143,28 @@ def test_lightning_kwargs(tmp_dir):
...
+    assert latest["step"] == 8
+    assert latest["epoch"] == 1
+
+    scalars = os.path.join(
+        dvclive_logger.experiment.plots_dir, Metric.subfolder
+    )
+    epoch_loss = history[os.path.join(scalars, "train", "epoch", "loss.tsv")]
+    step_loss = history[os.path.join(scalars, "train", "step", "loss.tsv")]
+    assert len(epoch_loss) == 2
+    assert len(step_loss) == 4
```
Comment (daavoo) [inline tests/test_lightning.py:170]: I have no idea what are the differences between these 2. 
I assume `epoch/metric` contains the mean value of the whole epoch 🤷

**Review by daavoo — APPROVED** [review]

File: tests/test_lightning.py (hunk 1 above)
Comment (daavoo) [reply tests/test_lightning.py:170]: Found https://pytorch-lightning.readthedocs.io/en/stable/extensions/logging.html#automatic-logging
